deliverables/README.md

```diff
@@ -9,60 +9,123 @@ To answer questions, please just put the answer on the next line after the quest
 What is Git?  
+Git is a version Control system that helps keep track of changes in one's code.
+
 What is SCM?  
-What is a VCS?  
+SCM stands for "Supply Chain Management" which is the oversight of information, as they move in a process from supplier to developer and to userer.
+What is a VCS? 
+A version control system is a tool that a developer or group of developers use to keep track of changes with in a project.
+
 Why is Git useful for a developer?  
+Git allows a developer to track their changes in a program and experiment without messing up their working project.
+
 Why is Git useful for a team of developers?  
+Git allows a team of developers to track changes with in their document. It also useful if one wanted to have seperate team members modify certain features in an application. For example they can build seperate branches so they can experiment with the code without messing with the running product. 
+
 How do you create a new Git repository for a project locally?  
+"git init"
+
 How do you create it on Github?  
+Next to your username click the "+" icon and then click the "new repository" option.
+
 How do you commit changes?  
+"git commit"
+
 What is the difference between staging and committing changes?  
+Staging changes is a metaphorical loading dock where all your changes get shiped away by eventually commiting them. Where as commit records all changes to the repository. Thus staging gives you alot more flexibility to what you want to change specifically. 
+
 What is the difference between committing your changes and pushing them to Github?  
+Commiting your changes only saves them to your local repository where pushing them Github allows your team to expierment with your project
+
 What is the command to check the status of your current repo in git?  
-What is the command to see the history of your previous commits (from the command line)?  What are a few interesting variations (sets of options)?  
+"git status"
+
+What is the command to see the history of your previous commits (from the command line)?  What are a few interesting variations (sets of options)? 
+"git log" Shows commit logs.
+"git log -follow" Continue listing the history of a file beyond renames (works only for a single file).
+"git log -log-size" Include a line “log size <number>” in the output for each commit, where <number> is the length of that commit’s message in bytes. 
+
 How can you look through your historical commits on the Github website?  
+"http://github.com/<username>/<project>/commits/<branch>/"
+<username> is the username of the person that owns the repo
+<project> is the repo name
+<branch> can be 'master' or any other branch
+
 What is a "Merge"?  
-What is a "Pull Request"?  
-What is "Forking" a repo?  
+Merging is Git's way of putting a forked history back together again. The git merge command lets you take the independent lines of development created by git branch and integrate them into a single branch.
```

Comment:
I notice this is copy-pasted from https://www.atlassian.com/git/tutorials/using-branches/git-merge.  In the future if you want to do this you should link to the source.  

Also if you were to re-form this into your own words it would enhance your understanding.  Obviously you are the one to budget your own time so it's your call.